Manage Logon Hour via Windows PowerShell

```diff
@@ -0,0 +1,345 @@
+---
+post_title: Manage Logon Hour via Windows PowerShell
+username: farisnt@gmail.com
+Catagories: PowerShell, Active Directory
+Summary: How to change users Logon Hours using PowerShell
+---
+
+
+**Q:** How can I set AD domain user Logon Hour via Windows PowerShell ?
+
+**A:** Most of the Active Directory object properties can be easily read or modified by using the `Get-` or `Set-` with the property name and its value, but when it comes to **LogonHours**, it not that easy. There are multiple steps and challenges required to build the value, and in this tutorial, you will learn all the tips and tricks needed to set the **LogonHours** hours via Windows PowerShell. So get a cup of coffee, and let's start.
+
+## Logon Hours, Understanding the basics.
+
+First, let's start with the GUI of the Logon Hours property by opening any AD user account property —> **Account** Tab —> click on **Logon Hours.** The default value is all blue, which represents logon permitted. If there is any white block, it represents a logon denied on that selected time/day slot.
+
+In the example below, the user cannot log in on Sunday or Saturday all day but can log in from Monday through Friday. 
+
+![Manage%20Logon%20Hour%20via%20Windows%20PowerShell%20c28909d216f44708aad42cfa6706b403/Untitled.png](Manage%20Logon%20Hour%20via%20Windows%20PowerShell%20c28909d216f44708aad42cfa6706b403/Untitled.png)
+
+Logon Hours GUI
+
+Make to enable the Advanced Features in Active Directory Management Console to see the Attribute Editor for the users, open the Active Directory console, click on **View** Menu and select **Advanced Features**. 
```

Comment:
"Make to enable the Advanced Features!  should read "nable the Advanced Features..."